settings/provisioner/virsh/virsh.spec

```diff
@@ -8,28 +8,39 @@ subparsers:
                   host:
                       type: str
                       help: Address/FQDN of the BM hypervisor
+                      required: yes
                   ssh-user:
                       type: str
                       help: User to SSH to the host with
+                      default: root
```

Comment:
This will be added automatically. Check these line: https://github.com/obaranov/InfraRed/blob/RHOSINFRA-25/cli/spec.py#L198-L200